addons/web/models/base_document_layout.py

```diff
@@ -58,13 +85,17 @@ def _compute_logo_colors(self):
                 wizard_for_image = wizard
             wizard.logo_primary_color, wizard.logo_secondary_color = wizard_for_image._parse_logo_colors()
 
-    @api.depends('report_layout_id', 'logo', 'font', 'primary_color', 'secondary_color')
+    @api.depends('report_layout_id', 'logo', 'font', 'primary_color', 'secondary_color', 'report_header', 'report_footer', 'external_report_layout_id')
```

Comment:
Not sure about `'external_report_layout_id'`, could you explain?

Reply:
Can be removed, left over from the start of the task when I was trying to understand the code.